inst/tinytest/test_xptr.R

```diff
@@ -60,11 +60,9 @@ void test() {
 '
 
 writeLines(code, file_path)
-expect_silent(system(cmd), info="check that finalizer is NOT called on exit")
+expect_equal(system(cmd, intern=TRUE), character(0))
```

Comment:
Doh. I need better coffee!

Reply:
So did I when I wrote that in the first place! ;-)